UPGRADING.md

```diff
@@ -4,6 +4,74 @@ This guide provides instructions for upgrading to specific versions of Cosmos SD
 
 ## [Unreleased]
 
+### Refactor:remove bytes/HexBytes
```

Comment:
Nice! this section does sound a bit repetitive (as it repeats parts of other sections). Moreover there is already a section for unreleased, could you put it there?
Could the description be bit smaller and more generalized to all (custom) modules?

Reply:
ok, I will do it!